code/lib/cli/src/sandbox-templates.ts

```diff
@@ -641,6 +654,7 @@ export const daily: TemplateKey[] = [
   'preact-webpack5/default-js',
   'preact-vite/default-js',
   'html-vite/default-js',
+  'internal/react16-webpack',
```

Comment:
It's because the template has `inDevelopment: true`, it gets completely ignored. you need to remove that flag and then to the dance:
1. Changing the sandbox generator to point at this branch
2. Trigger the generator
3. Change the sandbox generator back again
4. Merge this before the daily generator kicks in

Reply:
This is what should have happened without the inDevelopment flag:
![image](https://github.com/storybookjs/storybook/assets/1671563/34435d79-4cd6-4a97-8096-f271fc3f43b9)

I added a small note to the generated message which will show the inDev templates, hopefully will make things less confusing!